README.md

```diff
@@ -7,24 +7,19 @@
 
 <!-- badges: end -->
 
-Shiny is a new package from RStudio that makes it incredibly easy to build interactive web applications with R.
-
-For an introduction and examples, visit the [Shiny Dev Center](https://shiny.rstudio.com/).
-
-If you have general questions about using Shiny, please use the [RStudio Community website](https://community.rstudio.com). For bug reports, please use the [issue tracker](https://github.com/rstudio/shiny/issues).
+Easily build rich and productive interactive web apps in R &mdash; no HTML/CSS/JavaScript required.
 
 ## Features
 
-* Build useful web applications with only a few lines of code&mdash;no JavaScript required.
-* Shiny applications are automatically "live" in the same way that spreadsheets are live. Outputs change instantly as users modify inputs, without requiring a reload of the browser.
-* Shiny user interfaces can be built entirely using R, or can be written directly in HTML, CSS, and JavaScript for more flexibility.
-* Works in any R environment (Console R, Rgui for Windows or Mac, ESS, StatET, RStudio, etc.).
-* Attractive default UI theme based on [Bootstrap](https://getbootstrap.com/).
-* A highly customizable slider widget with built-in support for animation.
-* Prebuilt output widgets for displaying plots, tables, and printed output of R objects.
-* Fast bidirectional communication between the web browser and R using the [httpuv](https://github.com/rstudio/httpuv) package.
-* Uses a [reactive](https://en.wikipedia.org/wiki/Reactive_programming) programming model that eliminates messy event handling code, so you can focus on the code that really matters.
-* Develop and redistribute your own Shiny widgets that other developers can easily drop into their own applications (coming soon!).
+* An intuitive and extensible [reactive](https://en.wikipedia.org/wiki/Reactive_programming) programming model which makes it easy to transform existing R code into a "live app" where outputs automatically react to new user input.
+  * This programming model naturally leads to more efficient and simpler code then the alternative (i.e., event-based programming) because, when Shiny receives new user input, it intelligently finds the minimal amount of work necessary to update output(s).
+* A prebuilt set of highly sophisticated, customizable, and easy-to-use widgets (e.g., plots, tables, sliders, dropdowns, date pickers, and more).
+* An attractive default look based on [Bootstrap](https://getbootstrap.com/) which can also be easily customized with the [bslib](https://github.com/rstudio/bslib) package or avoided entirely with more direct R bindings to HTML/CSS/JavaScript.
+* Seamless integration with [R Markdown](https://beta.rstudioconnect.com/content/2671/Combining-Shiny-R-Markdown.html), making it easy to embed numerous applications natively within a larger dynamic document.
```

Comment:
Not needed for this PR, but that doc should really be published somewhere more permannet/prominent.

Reply:
Yea, I'll use a more official/intro link instead https://shiny.rstudio.com/articles/interactive-docs.html